### PR TITLE
Create 1.7.7.4 release note

### PR DESCRIPTION
**Pull request by marionf**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Create 1.7.7.4 release note

### CODE REVIEW

File: news/_posts/releases/2021-03-30-prestashop-1-7-7-4-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ - 1.7.7.x
+ - 1.7
+---
+
+PrestaShop 1.7.7.4 is now available. This maintenance release fixes 13 issues reported since the 1.7.7.0.
+
+![1.7.7.4 is available!](/assets/images/2021/05/build_1.7.7.4.png)
+
+We suggest upgrading your shop quickly in order to benefit from these fixes. Of course, don't forget to backup before.
+
```
Comment (matks) [inline news/_posts/releases/2021-03-30-prestashop-1-7-7-4-maintenance-release.md:23]: Should we add
```suggestion

## Main fixes

```

File: news/_posts/releases/2021-03-30-prestashop-1-7-7-4-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+**Back-office**
+- ["Expand all" button of ps_facetedsearch not working](https://github.com/PrestaShop/PrestaShop/issues/23558)
+- [When a cart rule with a free gift and percentage discount is added to the cart, the discount amount displayed is incorrect](https://github.com/PrestaShop/PrestaShop/issues/22948)
+- [Product customization file can’t be downloaded if the customization label field contains a dot](https://github.com/PrestaShop/PrestaShop/issues/24036)
```
Comment (matks) [inline news/_posts/releases/2021-03-30-prestashop-1-7-7-4-maintenance-release.md]:
```suggestion
- [Product customization file cannot be downloaded if the customization label field contains a dot](https://github.com/PrestaShop/PrestaShop/issues/24036)
```

File: news/_posts/releases/2021-03-30-prestashop-1-7-7-4-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- [The gift block in the add order page is not updated when changing the carrier](https://github.com/PrestaShop/PrestaShop/issues/23751)
```
Comment (matks) [inline news/_posts/releases/2021-03-30-prestashop-1-7-7-4-maintenance-release.md]:
```suggestion
- [The gift block in the 'Add an order' page is not updated when changing the carrier](https://github.com/PrestaShop/PrestaShop/issues/23751)
```

File: news/_posts/releases/2021-03-30-prestashop-1-7-7-4-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title:  "Release of PrestaShop 1.7.7.4"
+subtitle: "Maintenance version of the 1.7.7.x branch"
+date:   2021-05-07 12:00:00
```
Comment (Progi1984) [inline news/_posts/releases/2021-03-30-prestashop-1-7-7-4-maintenance-release.md]:
```suggestion
date:   2021-05-03 12:00:00
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback on the date but it's ok for me

**Review by matks — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**